public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,188 @@
+# open()
+
+Opens a tracked file.
+
+```py
+dvc.api.open(path: str,
+             repo: str = None,
+             rev: str = None,
+             remote: str = None,
+             mode: str = "r",
+             encoding: str = None)
+```
+
+#### Usage:
+
+```py
+import dvc.api
+
+with dvc.api.open(
+        'get-started/data.xml',
+        repo='https://github.com/iterative/dataset-registry'
+        ) as fd:
+    # ... fd is a file descriptor that can be processed normally.
+```
+
+## Description
+
+Open file or model (`path`) tracked in a <abbr>DVC project</abbr> (by DVC or
+Git), and generate a corresponding
+[file object](https://docs.python.org/3/glossary.html#term-file-object).
+
+> The exact type of file object generated depends on the `mode` used. For more
+> details, please refer to Python's
+> [`open()`](https://docs.python.org/3/library/functions.html#open) built-in,
+> which is used under the hood.
+
+`dvc.api.open()` may only be used as a
+[context manager](https://www.python.org/dev/peps/pep-0343/#context-managers-in-the-standard-library)
+(using the `with` keyword, as shown in the **Examples** below).
+
+> Use `dvc.api.read()` to get the file's contents directly – no _context
+> manager_ involved.
+
+This function reads (streams) the file trough a direct connection to the storage
+whenever possible, so it does not require any space on the disc to save the file
+before making it accessible. The only exception is when using a Google Drive
+[remote type](/doc/command-reference/remote/add#supported-storage-types).
+
+## Parameters
+
+- **`path`** - location and file name of the file in `repo`, relative to the
+  project's root.
+
+- `repo` - specifies the location of the DVC project. It can be a URL or a file
+  system path. Both HTTP and SSH protocols are supported for online Git repos
+  (e.g. `[user@]server:project.git`). _Default_: The current project is used
+  (the current working directory tree is walked up to find it).
+
+- `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
+  a branch or tag name, or a commit hash). If `repo` is not a Git repo, this
+  option is ignored. _Default_: `HEAD`.
+
+- `remote` - name of the [DVC remote](/doc/command-reference/remote) to look for
+  the target data. _Default_: The
+  [default remote](/doc/command-reference/remote/default) of `repo` is used if a
+  `remote` argument is not given. For local projects, the <abbr>cache</abbr> is
+  tied before the default remote.
+
+- `mode` - specifies the mode in which the file is opened. Defaults to `"r"`
+  (read). Mirrors the namesake parameter in builtin
```

Comment:
in _the_ ?

Reply:
Referring to "in builtin"? Should be Ok like this actually. It's like saying "I live in building C."